installing/installing_openstack/installing-openstack-user-sr-iov.adoc

```diff
@@ -18,6 +18,7 @@ Using your own infrastructure allows you to integrate your cluster with existing
 * You verified that {product-title} {product-version} is compatible with your {rh-openstack} version by using the xref:../../architecture/architecture-installation.adoc#supported-platforms-for-openshift-clusters_architecture-installation[Supported platforms for OpenShift clusters] section. You can also compare platform support across different versions by viewing the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
 * Your network configuration does not rely on a provider network. Provider networks are not supported.
 * You have an {rh-openstack} account where you want to install {product-title}.
+* You understand performance and scalability practices for cluster scaling, control plane sizing, and etcd. For more information, see xref:../../scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc#recommended-host-practices[Recommended performance and scalability practices].
```

Comment:
The link points to a section named as `Recommended control plane practices` or possibly `Recommended practices for scaling the cluster`. For accessibility reasons your link anchor text should match one of these headings.

Reply:
```suggestion
* You understand performance and scalability practices for cluster scaling, control plane sizing, and etcd. For more information, see xref:../../scalability_and_performance/recommended-performance-scale-practices/recommended-control-plane-practices.adoc#recommended-host-practices[Recommended practices for scaling the cluster].
```